packages/lesswrong/lib/make_voteable.ts

```diff
@@ -149,19 +147,16 @@ export const makeVoteable = <T extends DbVoteableType>(collection: CollectionBas
     score: {
       type: Number,
       optional: true,
-      defaultValue: 0,
+      nullable: false,
       canRead: ['guests'],
-      onInsert: (document: DbInsertion<T>): number => {
-        // default to 0 if empty
-        return document.score || 0;
-      }
+      ...schemaDefaultValue(0),
     },
     // Whether the document is inactive. Inactive documents see their score
     // recalculated less often
     inactive: {
       type: Boolean,
       optional: true,
-      onInsert: () => false
+      ...schemaDefaultValue(false),
```

Comment:
We should double-check whether this substitution is safe, since it adds `onUpdate` as well as `onCreate` (and a couple other fields)